vizro-core/docs/pages/user_guides/table.md

```diff
@@ -0,0 +1,286 @@
+# How to use tables
+
+This guide shows you how to use tables to visualize your data in the dashboard.
+
+The [`Page`][vizro.models.Page] models accepts the `components` argument, where you can enter your visual content e.g.
+[`Graph`][vizro.models.Graph], [`Table`][vizro.models.Table], [`Card`][vizro.models.Card] or [`Button`][vizro.models.Button].
+
+## Table
+
+The [`Table`][vizro.models.Table] model allows you to visualize data in a tabular format.
+
+To add a [`Table`][vizro.models.Table] to your page, do the following:
+
+- insert the [`Table`][vizro.models.Table] model into the `components` argument of the
+[`Page`][vizro.models.Page] model
+- enter any of the currently available table functions
+
+See below for an overview of currently supported table functions.
+
+### Dash DataTable
+
+The [Dash DataTable](https://dash.plotly.com/datatable) is an interactive table component designed for viewing, editing, and exploring large datasets.
+
+You can use the [Dash DataTable](https://dash.plotly.com/datatable) in Vizro by importing
+```py
+from vizro.tables import dash_data_table
+```
+The Vizro version of this table differs in one way from the original table: it requires the user to provide a pandas dataframe as source of data.
+This must be entered under the argument `data_frame`.
+All other [parameters of the Dash DataTable](https://dash.plotly.com/datatable/reference) can be entered as kwargs.
+
+!!! example "Dash DataTable"
+    === "app.py"
+        ```py
+        import vizro.models as vm
+        import vizro.plotly.express as px
+        from vizro import Vizro
+        from vizro.tables import dash_data_table
+
+        df = px.data.gapminder().query("year == 2007")
+
+        page = vm.Page(
+            title="Example of a Dash DataTable",
+            components=[
+                vm.Table(id="table", title="Dash DataTable", figure=dash_data_table(data_frame=df)),
```

Comment:
Hm, I sometimes things it can be nice to be a bit explicit in examples, as this is often the starting point for modification. Someone might like to refer to the table id for Parameters eventually. I'll leave it for now